### DISCUSSION TITLE
Readability of Code Preview Links: Documentation Site

### DISCUSSION THREAD

**Discussion by bennettcc** [NCIOCPL/ncids#1379] [NCIDS Launch]
Submitted by Jorge during Doc site review:

 Not sure if there’s a particular reason for the color choices in the code imports (I think USWDS uses plain black). Contrast may be too low on the teal, hard to read for some users. May need to do a manual check.

This is applicable on component pages with code snippets

**Comment by blilianyu** [comment]
Code snippets theme has been changed before soft launch. There should not be any contrast issues anymore.

cc: @monika-jaeger

**Comment by laurelthrash** [comment]
This item has been addressed as referenced in the above comment. Closing this discussion. Cc: @blilianyu @monika-jaeger
